check.sh

```diff
@@ -175,7 +175,7 @@ function cmd_test() {
 
 function cmd_itest() {
     findGodot && \
-        run cargo build -p itest "${extraCargoArgs[@]}" && \
+        run cargo build -p itest "${extraCargoArgs[@]}" --features codegen-full,godot/api-custom && \
```

Comment:
This must be reverted -- `codegen-full` massively increases local test time, and switches between features for the different `check.sh` subcommands. Further, `api-custom` puts much more requirements on a user.